docs/docs/smart-contracts/PSP35/psp35.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 1
+title: PSP35
+---
+
+This example shows how you can reuse the implementation of [PSP35](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp1155) token. Also, this example shows how you can customize the logic, for example, to track the number of token types with `unique_ids`, adding a new token type with the `add_type` function.
+
+## Step 1: Include dependencies
+
+Include `brush` as dependency in the cargo file or you can use [default `Cargo.toml`](/smart-contracts/overview#the-default-toml-of-your-project-with-openbrush) template.
+After you need to enable default implementation of PSP35 via `brush` feature.
+
+```toml
+brush = { tag = "v1.7.0", git = "https://github.com/Supercolony-net/openbrush-contracts", default-features = false, features = ["psp1155"] }
```

Comment:
You need to adapt the documentation and use a new feature `psp35` instead of `psp1155`.